packages/react-button/src/components/MenuButton/MenuButton.scss

```diff
@@ -0,0 +1,5 @@
+@import '../Button/Button.scss';
+
+.menuIcon + * {
```

Comment:
i think  we should just do something like
```
.menu {
  margin: 0;
}
```

Reply:
This does not work, at least if we use `ContextualMenu`, because it is created in a separate `Layer`.